### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by andrew17752985**
https://andrew17752985.github.io/react_pagination/

### CODE REVIEW

File: src/components/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -1 +1,66 @@
...
+
+  const handlePageChange = (newPage: number) => {
+    if (newPage !== currentPage) {
+      onPageChange(newPage);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
```
Comment (maxim2310) [inline src/components/Pagination/Pagination.tsx:27]: or even better to use class names
```suggestion
      <li className={`page-item ${currentPage === 1 && 'disabled' }`}>
```

**Review by maxim2310 — APPROVED** [review]
Good work!!!

File: src/App.tsx [hunk 2]
```diff
@@ -34,65 +56,20 @@ export const App: React.FC = () => {
...
+      />
+
       <ul>
-        <li data-cy="item">Item 1</li>
-        <li data-cy="item">Item 2</li>
-        <li data-cy="item">Item 3</li>
-        <li data-cy="item">Item 4</li>
-        <li data-cy="item">Item 5</li>
+        {displayedItems.map((item) => (
+          <li key={uuidv4()} data-cy="item">
```
Comment (anastasiiavorobiova) [inline src/App.tsx:68]: it's better to use an item for keys, the UUID will generate a new key for each render

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

File: src/components/Pagination/Pagination.tsx [hunk 3]
```diff
@@ -1 +1,66 @@
...
+            data-cy="pageLink"
+            className="page-link"
+            href={`#${page}`}
+            onClick={() => handlePageChange(page)}
+          >
+            {page}
+          </a>
+        </li>
+      ))}
+      <li className={`page-item ${currentPage === totalPages && 'disabled'}`}>
```
Comment (etojeDenys) [inline src/components/Pagination/Pagination.tsx:53]: we can use classNames package for such cases

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**